runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -1,9 +1,12 @@
 name: Univocity Parsers
-#description: Univocity Parsers ...
+description: Univocity Parsers ...
```

Comment:
Mind improving this description a bit? :)

Reply:
Even better: Remove it and declare in the runtime's pom.xml's `<description>` element

Reply:
I did it